test/parallel/test-http-get-pipeline-problem.js

```diff
@@ -38,13 +39,11 @@ const image = fixtures.readSync('/person.jpg');
 console.log(`image.length = ${image.length}`);
 
 const total = 10;
-let requests = 0;
-let responses = 0;
+const requestCountdown = new Countdown(total, () => server.close());
+const responsecountdown = new Countdown(total, () => checkFiles());
```

Comment:
It is better to add a `mustCall` for these two lines. For example:
```js
new Countdown(total, common.mustCall(() => server.close()));
```

Reply:
Thanks. Its done!!!